task2.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+
+def multiply_digits_of_number(inputNum):
+    lst_digits = list(str(inputNum))
```

Comment:
You don't need to convert to list. Strings are also iterable. Also you can validate number type at the beginning.

```python
def multiply_digits(number):
    if not isinstance(number, int):
        raise TypeError('Must be an integer')
    
    product = 1
    for digit in str(number):
        product *= int(digit)

    return product
```